dataprofiler/profilers/profiler_options.py

```diff
@@ -695,14 +695,15 @@ def _validate_helper(self, variable_path='OrderOptions'):
 
 class CategoricalOptions(BaseInspectorOptions):
 
-    def __init__(self):
+    def __init__(self, is_enabled=True, top_k_categories=None):
```

Comment:
Hmmm, is this going to be the new pattern going forward? I see a lot of option inits with and without parameters so im unsure

Reply:
We should probably discuss this because there is currently no consistency (e.g. `DataLabelerOptions` sets everything to `None`, doesn't take parameters, `TextProfilerOptions` takes a bunch of optional parameters)

Reply:
The pattern started without. However, I believe I mentioned before it should be the pattern moving forward to include. As such we should in the future hopefully be able to init options via construction in one command rather than two.